src/File/utils.ts

```diff
@@ -88,6 +91,7 @@ export default function makeFileUtils(store: Store): FileUtils {
     getFileSelf: async (
       selfUrl: string,
       shouldFetchFile: boolean = false,
+      receiveAs?: HttpConfigTypes,
```

Comment:
I would make this an object (like options) in case we have to add more that one option. That way, we won't break the API